src/components/bs5/video/video.hbs

```diff
@@ -0,0 +1,41 @@
+<!-- QGDS Component: Video -->
+
+{{#if analyticsTrackingCode}}
+<script type="text/javascript" defer="defer" src="https://extend.vimeocdn.com/ga/{{analyticsTrackingCode}}.js">
+</script>
+{{/if}}
+
+
+<section class="video not-ready {{videoSize}}">
+    <div class="video-player ratio ratio-{{aspectRatio}}">
+        <a href="#" class="video-thumbnail" style="background-image:url({{thumbnail}})">
+
+            <div class="video-nav">
+                <div class="video-watch"><span class="icon"></span><span>Watch</span></div>
+
+            {{#if duration}}
+                <div title="Video duration" class="video-duration"><span class="icon"></span><span>{{duration}}</span></div>
+            {{/if}}
+            </div>
+        </a>
+
+        <div class="video-embed ratio ratio-{{aspectRatio}}">
+            {{#ifCond source '===' 'vimeo'}}
+                <iframe title="Vimeo Video" class="embed-responsive-item video-vimeo" allow="autoplay; fullscreen" allowfullscreen muted="muted" src="https://player.vimeo.com/video/{{videoId}}?rel=0"></iframe>
+
+            {{else}}
```

Comment:
We should not play favourites on hosting providers. As such;

Can you make it else if === 'youtube', 


else if === 'custom', else (no video provide error)



then surface the extra items for custom video inclusion 
```
{{#ifCond source '===' 'custom'}}
                <iframe title="Custom Video" class="embed-responsive-item video-custom" allow="{{videoAllowParams}}" allowfullscreen muted="muted" src="{{videoIframeUrl}}{{videoId}}?{{videoIframParams}}"></iframe>

            {{else}}
No Video system selected
{{/ifcond}}
```
future idea: add gtm4 metrics collecting stats custom is used and if a new option to this component should be introduced.